coderd/database/drivers.go

```diff
@@ -24,3 +24,22 @@ func (a *Actions) Scan(src interface{}) error {
 func (a *Actions) Value() (driver.Value, error) {
 	return json.Marshal(a)
 }
+
+// TemplateACL is a map of ids to permissions.
+type TemplateACL map[string][]rbac.Action
```

Comment:
Could this be a `map[uuid.UUID][]rbac.Action`?

Reply:
possibly but it involves a lot of code changes, i'll try it out in a separate pr